app/feature/feature-edit-coinsured/src/main/kotlin/com/hedvig/android/feature/editcoinsured/ui/AddCoInsuredBottomSheetContent.kt

```diff
@@ -226,6 +233,37 @@ private fun FetchFromSsnFields(
   }
 }
 
+private fun personNummerInputFormatter(text: AnnotatedString, mask: String): TransformedText {
+  val trimmed = if (text.text.length >= 12) text.text.substring(0..11) else text.text
+
+  val annotatedString = AnnotatedString.Builder().run {
```

Comment:
I think you can use the `val annotatedString = buildAnnotatedString { ...` builder here instead. Just a bit nicer syntax of the same thing.